megamek/src/megamek/client/ui/swing/AccessibilityWindow.java

```diff
@@ -38,12 +38,18 @@ public AccessibilityWindow(ChatterBox cb, ClientGUI clientgui) {
         client.getGame().addGameListener(new GameListenerAdapter() {
             @Override
             public void gamePlayerConnected(GamePlayerConnectedEvent e) {
-                systemEvent("New player has connected. Their name is " + e.getPlayer().getName() + ".");
+                String name = e != null && e.getPlayer() != null 
```

Comment:
This might be easier to read if it is in brackets?

Reply:
I don't usually use brackets in simple cases.